src/foam/core/types.js

```diff
@@ -226,6 +226,12 @@ foam.CLASS({
 
         return foam.Date.compare(o1, o2);
       }
+    },
+    {
+      name: 'format',
+      value: function(val) {
+        return foam.Date.formatDate(val);
```

Comment:
How does this not display the time?

Reply:
because of change from [a previous commit](https://github.com/kgrgreer/foam3/pull/863/commits/0c4673dfcd2825510f028cb71b719e3d180b75e5), so now the time is only displayed when a second argument is provided

Reply:
Please update the doc, this is very subtle.